### PR TITLE
Balder/allow chunks larger than 1k

### PR DESCRIPTION
**Pull request by baldersheim**
@jonmv or @bjorncs or @vekterli PR

### CODE REVIEW

**Review by jonmv — APPROVED** [review]

**Review by vekterli — COMMENTED** [review]
Removing the upper bound for the amount of memory that can be buffered during visiting disables the safety of a rather large foot-gun—but at least this is a case where the caller explicitly asks for it to happen.

File: vespaclient-container-plugin/src/main/java/com/yahoo/document/restapi/resource/DocumentV1ApiHandler.java [hunk 1]
```diff
@@ -1168,9 +1148,8 @@ private void updateRemoveMetrics(Outcome outcome, double latency) {
     // ------------------------------------------------- Visits ------------------------------------------------
 
     private VisitorParameters parseGetParameters(HttpRequest request, DocumentPath path, boolean streamed) {
-        int wantedDocumentCount = Math.min(streamed ? Integer.MAX_VALUE : 1 << 10,
-                                           getProperty(request, WANTED_DOCUMENT_COUNT, integerParser)
-                                                   .orElse(streamed ? Integer.MAX_VALUE : 1));
+        int wantedDocumentCount = getProperty(request, WANTED_DOCUMENT_COUNT, integerParser)
+                .orElse(streamed ? Integer.MAX_VALUE : 1 << 10);
```
Comment (jonmv) [inline vespaclient-container-plugin/src/main/java/com/yahoo/document/restapi/resource/DocumentV1ApiHandler.java:1152]: You've also changed the default from 1 to 1024 here. That's probably OK? But we need to update some system test if we want this change.

Comment (baldersheim) [reply vespaclient-container-plugin/src/main/java/com/yahoo/document/restapi/resource/DocumentV1ApiHandler.java:1152]: I noticed that too, will reconsider.

**Comment by baldersheim** [comment]
@vekterli I removed the limit, as I wanted to see impact on performance stability.
Will see what makes sense afterwards.

**Review by bjorncs — COMMENTED** [review]
👍

**Pull request merged**